docs/proposals/kubeedgeadm-scope.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Kubeedgeadm kubeEdge installer scope (Issue 324)
```

Comment:
kubeedgeadm ? the installer commands only have kubeedge in it .

Reply:
nit: kubeEdge -> KubeEdge
Applicable to similar places below